src/test/resources/Tests/cases/CaseServiceTest/getCasesOk1/expectedResponse.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "courthouse": "SWANSEA",
+    "courthouse": "case_courthouse",
```

Comment:
the comment is for the folder name - getCasesOk1. Any reason for a numeric folder name?

Reply:
'getCasesOk1' is the name of the test

Reply:
I don't think this is a good naming convention so I actually changed the names of the test to reflect that. Can you rename them so it makes more sense.  Maybe something like expectedResponseForMultipleCases and expectedResponseForSingleCase.

Reply:
renamed folders to match tests